### PR TITLE
Use keyword search for up to 2 words

### LINKED ISSUE #215: Use keyword search for very short search phrases (1-2 words)
People were searching for `corrigible` and `prosaic` and not finding results because sentence embeddings don't match single words semantically very well => let's use plaintext search for 1-2 words.

### PR DESCRIPTION
**Pull request by Aprillion**
* fix #215
* don't immediately post suggestion when there is error fetching extra questions
* show `no results` message when plaintext search returns empty array
plaintext search

* [ ] merge PR #216 before this one (target branch or this PR will automatically update to `master` when the other one is merged first)

### CODE REVIEW

File: app/components/search.tsx [hunk 1]
```diff
@@ -70,26 +71,33 @@ export default function Search({onSiteAnswersRef, openQuestionTitles, onSelect}:
     initWorker()
   }, [])
 
-  const searchFn = (value: string) => {
+  const searchFn = (rawWalue: string) => {
+    const value = rawWalue.trim()
```
Comment (Aprillion) [inline app/components/search.tsx:75]: dammit typo, one moment please...

Comment (Aprillion) [reply app/components/search.tsx]: fixed

File: app/components/search.tsx [hunk 2]
```diff
@@ -222,26 +232,28 @@ const ShowMoreSuggestions = ({
...
     }
     getResults(question)
-  }, [setExtraQuestions, question])
+  }, [question])
 
   if (extraQuestions === empty) {
     return (
       <Dialog onClose={onClose}>
         <div className="loader"></div>
+        {error && <div className='error'>{error}</div>}
```
Comment (mruwnik) [inline app/components/search.tsx:256]: should this show the loader on errors? Or just an error message?

**Review by mruwnik — APPROVED** [review]

File: app/components/search.tsx (hunk 2 above)
Comment (Aprillion) [reply app/components/search.tsx:256]: there should be no errors, so doesn't matter IMHO 😅

**Pull request merged**